SchemaInventory.json

```diff
@@ -1208,6 +1208,20 @@
       "date": "8/13/2020",
       "dynamic": "No",
       "approved": "Yes"
+    },
+    {
+      "name": "ProcessPidGraphical",
+      "path": "Domains\\4-Application\\Plant\\Released\\ProcessPidGraphical.01.00.03.ecschema.xml",
+      "released": true,
+      "version": "01.00.03",
+      "comment": "",
+      "verifier": "",
```

Comment:
Can you please add a `verifier`? and then make the `verified` and `approved` fields to `yes`.

Reply:
@diegoalexdiaz Can you please be the "verifier" for this PR?  If you agree, I'll update the SchemaInventory.json with your name and a "yes" in the verified field.